internal/runtime/client/client.go

```diff
@@ -172,6 +172,39 @@ func (c *client) Unregister(extensionConfig *runtimev1.ExtensionConfig) error {
 	return nil
 }
 
+func (c *client) GetAllExtensions(ctx context.Context, hook runtimecatalog.Hook, forObject metav1.Object) ([]string, error) {
```

Comment:
Could be also a follow up, but I think that we should use GetAllExtensions to simplify/dedup code in CallAllExtensions

Reply:
Added to the umbrella issue

Reply:
Thank you